ProcessLib/RichardsComponentTransport/CreateRichardsComponentTransportProcess.cpp

```diff
@@ -43,30 +44,80 @@ std::unique_ptr<Process> createRichardsComponentTransportProcess(
         !(staggered_scheme && (*staggered_scheme == "staggered"));
 
     // Process variable.
-
     //! \ogs_file_param{prj__processes__process__RichardsComponentTransport__process_variables}
     auto const pv_config = config.getConfigSubtree("process_variables");
-
     std::vector<std::vector<std::reference_wrapper<ProcessVariable>>>
         process_variables;
+    auto range =
+        //! \ogs_file_param{prj__processes__process__RichardsComponentTransport__process_variables__process_variable}
+        pv_config.getConfigParameterList<std::string>("process_variable");
+
     if (use_monolithic_scheme)  // monolithic scheme.
     {
-        auto per_process_variables = findProcessVariables(
-            variables, pv_config,
+        std::vector<std::reference_wrapper<ProcessVariable>>
```

Comment:
Again, I think you can use functionality from `ProcessLib/Utils/ProcessUtils.cpp` that was used in the previous implementation.

Reply:
See the reply at the top.